src/components/App.jsx

```diff
@@ -185,6 +186,17 @@ export default function App() {
             )}
           </div>
           <div className="d-flex flex-1 order-2 justify-content-end align-items-center gap-1">
+            <div className="warning-hint text-start text-secondary d-none d-md-block gap-1 pe-1">
```

Comment:
```suggestion
            <div className="warning-hint text-start text-secondary d-none d-md-block pe-1">
```

This doesn't do anything I think (only applies to `d-flex`).